data/src/main/java/com/photosurfer/android/data/remote/model/request/LoginRequest.kt

```diff
@@ -0,0 +1,12 @@
+package com.photosurfer.android.data.remote.model.request
+
+import com.google.gson.annotations.SerializedName
+
+data class LoginRequest(
+    @SerializedName("socialToken")
+    val socialToken: String,
+    @SerializedName("socialType")
+    val socialType: String,
+    @SerializedName("fcm")
+    val fcm: String
```

Comment:
```suggestion
    @SerializedName("fcm")
    val fcmToken: String
```

먼가 로그인 requst니까 당연히 token이겠지만,
추후에 명확한 이해를 위해 `fcmToken`이라고 해주는것도 좋을듯!?!